### ISSUE TITLE
GeoJsonLayer rendering incorrectly

### ISSUE THREAD

**Issue by mohitshah9920** [uber/deck.gl#3892]
Hi deck.gl team,

We are trying to render geojson data using the GeoJsonLayer and it looks ruptured in the middle:

Here is Canada:
<img width="561" alt="canada" src="https://user-images.githubusercontent.com/3580703/68949968-2a39c600-07f6-11ea-9c40-1863ad3d99d3.png">

Here is how to replicate: https://codesandbox.io/s/deckgl-geojson-artifact-ycylr

It doesn't seem to happen to all geojsons though. 

Here is one more:
Europe
<img width="659" alt="europe" src="https://user-images.githubusercontent.com/3580703/68950101-6cfb9e00-07f6-11ea-9c78-4691c4e10e55.png">

[europe.txt](https://github.com/uber/deck.gl/files/3852572/europe.txt)

 - Framework Version: happening on 7.1.7, also on 7.3.6
 - Browser Version: Chrome 78.0.8
 - OS: Mac OS X 10.15

Thank you

**Comment by Pessimistress** [comment]
deck.gl performs triangulation on raw positions (lnglat in this case) instead of projected positions for perf reasons. It is usually sufficient unless you have a polygon that has a large span latitude-wise. In web mercator projection, shapes are stretched as latitude increase.

One solution for this is to allow the app opt-in to a "high precision" mode that triangulates more accurately but slower. Since you are working with static map data it should be a one-time cost.

**Comment by ibesora** [comment]
I'm seeing the same problem seen here and in #3746 .
I'm rendering a geojson polygon (a full-screen quad) with a hole and deck.gl is rendering this:
![image](https://user-images.githubusercontent.com/2301378/70802341-84a45300-1db1-11ea-9887-c370fc848369.png)
Mapboxgl is rendering this: 
![image](https://user-images.githubusercontent.com/2301378/70802378-9dad0400-1db1-11ea-9aa9-c7bc15f1a931.png)

The geojson is here: 
```javascript
{"type":"FeatureCollection","features":[{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[-59.43300005000003,41.706990219176745],[-145.80800005000003,41.706990219176745],[-145.80800005000003,3.556313377691388],[-59.43300005000003,3.556313377691388],[-59.43300005000003,41.706990219176745]],[[-99.84322258736285,27.76986281894883],[-99.93268683539574,27.67442073783978],[-99.99884482790605,27.61527940260827],[-100.0160769829773,27.63273257341551],[-100.1884255257484,27.79913718644218],[-100.2861798240665,27.73094725207428],[-100.3506098416027,27.68459855939462],[-100.4176995527788,27.39363275976545],[-100.6780062167529,27.33479730686584],[-100.8341234173161,27.19517396512464],[-100.8305613951266,27.07388547381954],[-100.8149102552778,27.07112377136633],[-100.7994361239934,27.04583075566871],[-100.7842299950126,27.0589222883931],[-100.7626740785359,27.06263413812065],[-100.740892063531,27.04105702932372],[-100.7360387056382,27.04919756130755],[-100.7467957340541,27.06430621401836],[-100.7486209809952,27.08125280937244],[-100.7247051661927,27.07788387593526],[-100.7093258721329,27.09876347694539],[-100.6946437331589,27.1097701529044],[-100.6767912667636,27.10687737816482],[-100.6747517596207,27.0864643075931],[-100.6537042318563,27.07572968896877],[-100.5397842752349,27.0474058299277],[-100.5360116899697,27.02535347749293],[-100.5406434813267,27.01489872909188],[-100.539683641452,26.98666999824864],[-100.5449924955915,26.97979155411805],[-100.5382393979423,26.93971985172625],[-100.544926288236,26.92534880779618],[-100.5384298025916,26.90611970805944],[-100.540665807366,26.88096998305062],[-100.5513586644356,26.85872973657332],[-100.5456165817789,26.8414470244184],[-100.5417727996939,26.8060015336133],[-100.5901358303338,26.7595206676225],[-100.601307130268,26.73707707842846],[-100.6144001987792,26.68563889035363],[-100.6912288888146,26.61378942300905],[-100.7530354398283,26.73520780235005],[-100.9050911692638,26.60863811413943],[-101.0788623890135,26.46926142919431],[-101.2067627100116,26.36977820668862],[-101.1854420546275,26.34540096063002],[-101.1045314974731,26.26162470241688],[-101.0917389592771,26.24451104504575],[-101.0746469196188,26.19897473415003],[-101.0473981916479,26.16984602261127],[-101.0088827646324,26.15382364448407],[-100.9827420438343,26.1394772064303],[-100.9428980179134,26.08678906824759],[-100.9164156670706,26.07561055513152],[-100.9046060999748,26.00870223483438],[-100.905531639878,25.95694570101114],[-100.8768786671691,25.94721916584676],[-100.8344248771625,25.94504290744105],[-100.8226689450481,25.91688454268897],[-100.8371196470295,25.86381259058213],[-100.8470687129401,25.85788573015099],[-100.8619013946318,25.81847861941758],[-100.825123594619,25.7517211165352],[-100.8056139470353,25.72430180397109],[-100.8036987069439,25.70875537747489],[-100.7877752830842,25.68359721286198],[-100.7811986511636,25.63858208214882],[-100.7297728999679,25.63389565999407],[-100.7199164705898,25.60306696033793],[-100.6902863444204,25.56710078559861],[-100.6329018439378,25.57129935014458],[-100.6119597625326,25.55139936666779],[-100.5963481241311,25.54448255411341],[-100.5384359056359,25.53185391961317],[-100.5296993302084,25.51439287860562],[-100.5997503423278,25.51995952743107],[-100.6190905896576,25.5056910845555],[-100.5795103081819,25.49454398410147],[-100.5848890867168,25.47081303893365],[-100.5609671288884,25.46453840334124],[-100.530883148633,25.46346765001757],[-100.5227030889345,25.4343247293081],[-100.5016903829274,25.43283676443727],[-100.4693400825134,25.42172174549766],[-100.45195490516,25.40811568507108],[-100.4700988220533,25.33778984496514],[-100.4460286219784,25.33185769637181],[-100.3945674167729,25.32785441682805],[-100.3268037028985,25.34035647140235],[-100.2815796173169,25.31971372370404],[-100.2639745015676,25.30529339672321],[-100.2541384345034,25.30690877171312],[-100.2431419024004,25.26744334962275],[-100.2625101250344,25.25402279360681],[-100.2344205076778,25.23549642894272],[-100.2295755984041,25.21791651217156],[-100.2613835064078,25.20707056795443],[-100.3052836105121,25.21341722337707],[-100.3255501172658,25.22225875875032],[-100.3320074140356,25.18628521147295],[-100.4074091922911,25.20896811143805],[-100.4436414296108,25.22314521762923],[-100.4985464678563,25.23750748119714],[-100.5296068814061,25.24014293452175],[-100.591962417472,25.25505001482155],[-100.7088877858605,25.23645757293063],[-100.8031304535166,25.16525846515781],[-100.8215055946887,24.99671083717765],[-100.7920630858357,24.97914818163081],[-100.7914630014435,24.97405067495767],[-100.7274954958694,24.9187165487326],[-100.8261324179632,24.83500457301935],[-100.8274134995556,24.82420788996465],[-100.7969782429431,24.75080690162261],[-100.8054729365843,24.59634011826138],[-100.7999341158552,24.55618607714999],[-100.8104163611759,24.55159378354945],[-100.8478056419007,24.55561860264763],[-100.8828144497271,24.57161489315055],[-100.9022337847306,24.58391773552375],[-100.9587926057055,24.54268406530075],[-101.0804951331809,24.5911853152009],[-101.1276196012299,24.62336176714113],[-101.1646459411706,24.6815589627532],[-101.1790350912053,24.73295667682686],[-101.2017255927346,24.76548832294486],[-101.2150391341662,24.77516784541729],[-101.2360889114732,24.77787896778077],[-101.3131231384611,24.80141085526373],[-101.3421415745856,24.81335250833335],[-101.4021570653302,24.78822243508848],[-101.4126549600933,24.77205500954783],[-101.4616393518874,24.73675681572369],[-101.4917053307349,24.7437758583297],[-101.5405787585103,24.74558125124316],[-101.6222697134612,24.75898308184484],[-101.640767225504,24.80070354448889],[-101.6090116254187,24.78966414003504],[-101.6012606305623,24.83556319447102],[-101.6563317655578,24.86079317405246],[-101.6851494911706,24.84001266790161],[-101.773462956643,24.8755351211375],[-101.8318984555903,24.97217322082578],[-101.8488100006271,25.0176236152333],[-102.0818056250298,25.07070060330675],[-102.2376222049235,25.11846237978269],[-102.2955130238811,25.12523550695677],[-102.463325206585,25.11591615452599],[-102.4467634443429,24.99951540838258],[-102.6696939820295,25.06663649202801],[-102.8531307178535,24.77200094651511],[-102.8564027915653,24.7397739649494],[-102.8832809075964,24.750584182082],[-102.8954320981794,24.76599359033764],[-102.9236292742289,24.78036285546743],[-102.9491447567931,24.7797059423143],[-102.9786998717824,24.79938425189624],[-103.006389685079,24.79358405671364],[-103.0372874881889,24.81144050348585],[-103.0539547930095,24.80998922714506],[-103.0676400704956,24.81803853850519],[-103.0967919591136,24.8265318905711],[-103.2591458990252,24.96465877549912],[-103.2453637785818,24.98246279556333],[-103.2466747512362,24.99641317172533],[-103.2740341692548,25.02090249801113],[-103.2668946099247,25.05548874018293],[-103.3326090987676,25.09645522877295],[-103.3460743190998,25.11068121835583],[-103.3595393064555,25.11002885056517],[-103.3768071945957,25.14150791716351],[-103.3973987150177,25.16385261264542],[-103.4033741423519,25.18263772715807],[-103.4442615274125,25.20869336963106],[-103.4492289392701,25.22711685705008],[-103.4728578723116,25.24977729833688],[-103.4967201013082,25.25574851789766],[-103.5098529183934,25.29007723526732],[-103.4985102552564,25.29412783329063],[-103.4964559137023,25.30928893740518],[-103.4790714412936,25.32499336684695],[-103.4864614245375,25.33075067519513],[-103.4701319963572,25.34617870704434],[-103.4618185709121,25.37637115765527],[-103.4069194671057,25.39889943473587],[-103.3804617300882,25.39095643914843],[-103.3655194336368,25.3931562754618],[-103.3510365540014,25.3728856510443],[-103.3369161956232,25.37123114588459],[-103.3413677237949,25.41014222896917],[-103.3690620025061,25.41646072713566],[-103.3906512869075,25.43459115712599],[-103.398747642523,25.45060038013557],[-103.4357285512358,25.47907398346356],[-103.4935117136705,25.53697217009721],[-103.4686155130523,25.54692377897828],[-103.4546535483488,25.5579287161979],[-103.4526263079402,25.57527906105589],[-103.4353665371402,25.582777544248],[-103.4249540255538,25.61998563899792],[-103.40747844809,25.62568145368363],[-103.4087358039667,25.63967215614543],[-103.3836379679057,25.65325395405018],[-103.3633923912487,25.6742469563314],[-103.3528360610087,25.71655308551082],[-103.3402891883523,25.72403447145262],[-103.326393421991,25.76044549319721],[-103.3189177895211,25.81480152019424],[-103.3463601017652,25.86030438362788],[-103.3504882178898,26.14238995872759],[-103.2608252628911,26.28472681884862],[-103.3167926008671,26.36177618432137],[-103.3741834918072,26.60235488778014],[-103.6369629174305,26.69256491014784],[-103.6438914521961,26.69015849194717],[-103.6582559985328,26.71948395351802],[-103.6990941045426,26.72525637705647],[-103.7152486990224,26.73507726057089],[-103.7248708067861,26.77210651552799],[-103.750619731675,26.80990508246411],[-103.7539679295032,26.8295734426893],[-103.7796054404329,26.83612625087076],[-103.8044780197223,26.85500959408527],[-103.8225349318971,26.8753743429061],[-103.8517007148073,26.89613383376826],[-103.8519434666723,26.90523044422502],[-103.8359191861112,26.93581334880716],[-103.8520890333017,26.93461942014532],[-103.7846287526091,26.9806985297971],[-103.7799476889131,27.00888629877625],[-103.7447891590093,27.02273114841179],[-103.751832863289,27.07665547234547],[-103.7834703244668,27.09120559451492],[-103.799398469589,27.12472953252595],[-103.7984273357073,27.21347568854459],[-103.8093954378473,27.24920308145195],[-103.8197634940107,27.24295655390931],[-103.8305932639319,27.27684748011934],[-103.8321225590804,27.29829347577866],[-103.8768760198798,27.31159213809497],[-103.8636504116585,27.44092074512224],[-103.8585051405959,27.43989149711995],[-103.8546009477036,27.54943273531416],[-103.8688376547967,27.60713682981465],[-103.8835555259651,27.60077175368316],[-103.8972628674039,27.65654877432038],[-103.8895049397126,27.71321714443676],[-103.8966922867911,27.74055724264885],[-103.8811453402529,27.73918574314805],[-103.9356495526254,27.7975527395856],[-103.9008439285959,27.82685725492917],[-103.9439067033651,27.82687055464338],[-103.9600019200556,27.83530901470973],[-103.9457830457564,27.85638822984184],[-103.9433312765744,27.88884322182071],[-103.9549734960707,27.95082099436762],[-103.9356430899815,27.94534236580189],[-103.9418675898997,27.92779649112745],[-103.8928858448253,27.91402296219836],[-103.8777159377725,27.95749033366409],[-103.9266918956438,27.97135675960366],[-103.889485248704,28.06037692545016],[-103.8609412983136,28.10902193243336],[-103.8674998614881,28.11213567359972],[-103.7681797793741,28.30037252472169],[-103.7628712390615,28.29819678868919],[-103.7283328636569,28.35649101366346],[-103.6430107967072,28.5124618707315],[-103.6479444617054,28.53298348361623],[-103.6245675938348,28.58114651423063],[-103.601008392371,28.57490740462847],[-103.5919158746046,28.59083425957872],[-103.6160460325457,28.59819473003268],[-103.5963737035045,28.63947454727528],[-103.5820014858454,28.62879351253474],[-103.5726589605388,28.64063917105657],[-103.5572953111297,28.61128067390748],[-103.5197619892688,28.66874502686574],[-103.5145121437414,28.66499784490751],[-103.4632546030118,28.7566145388328],[-103.4663453466946,28.75945763424082],[-103.4316630395004,28.81318034729641],[-103.410015390116,28.84092239094876],[-103.3942297882713,28.84011763384581],[-103.4036502507826,28.86925417131467],[-103.357700664759,28.92025062834719],[-103.306961888072,29.003925418497],[-103.2664797305823,28.99592468321693],[-103.2503390820911,28.98022171057544],[-103.2039463725611,28.9870934410815],[-103.1726006686617,28.98052574243512],[-103.1529999623594,28.97226482765144],[-103.1359324974961,28.98328336612094],[-103.1158546045633,28.98435727209538],[-103.1175169685243,28.99997004053258],[-103.0982097168101,29.02562503714659],[-103.1066478731668,29.03866086216558],[-103.1007096608698,29.06033087968991],[-103.0769069768994,29.07595338012479],[-103.0803310518592,29.08711973671059],[-103.0547010669193,29.10072449511695],[-103.0337972692943,29.10120396333264],[-103.0233169363653,29.12294286859156],[-103.0121995515522,29.12845424283763],[-103.0081789939701,29.1482492835302],[-102.9962593780218,29.16028062325546],[-102.9982749368919,29.17399137780626],[-102.9781195043657,29.18652350224794],[-102.9509042081749,29.17831950484215],[-102.9330229454459,29.19450937584169],[-102.9181354004998,29.19076533835091],[-102.9094706250674,29.21997360446862],[-102.8943699472455,29.20857287037022],[-102.8680148829549,29.2228623165564],[-102.8711974755471,29.24119115434552],[-102.8876163176759,29.2456990451696],[-102.906299561557,29.26207752427226],[-102.9034764473028,29.2783939148711],[-102.8896404306336,29.28865547914109],[-102.8875210542141,29.31118937312993],[-102.8770145086936,29.35484952117038],[-102.8633879045802,29.35132077622812],[-102.8392197688886,29.35889534666833],[-102.8449367751574,29.37430828137618],[-102.820147901515,29.4036716444885],[-102.832147189367,29.43232289834872],[-102.8253770161512,29.45793813400953],[-102.8143965290135,29.47128766492978],[-102.8072618125775,29.49423211226804],[-102.8077694369303,29.52309449945268],[-102.7782879578171,29.54360519922229],[-102.7739304175325,29.57323826161554],[-102.7620758317864,29.57997919162633],[-102.7684946967321,29.59480892746957],[-102.7401019074319,29.59807073237183],[-102.7410214316759,29.63586373042078],[-102.7257222074824,29.64326110499272],[-102.7036249390709,29.67159165834699],[-102.6986225839638,29.69734915780425],[-102.6889780266338,29.70502312216714],[-102.6910314418383,29.72200555888285],[-102.6769939576594,29.74157808547318],[-102.6462294811429,29.73381854098315],[-102.6227323744889,29.7361629617225],[-102.6149658138846,29.74647921389956],[-102.5886349470693,29.74911682638671],[-102.5632170961347,29.76789546053553],[-102.5483987286691,29.74458614348446],[-102.5138746184868,29.76553447944853],[-102.5167918500646,29.78446911815103],[-102.4803488534199,29.77611942622107],[-102.4662426446282,29.78377779495224],[-102.4576731618407,29.77483685644923],[-102.4339423390734,29.77705690679305],[-102.4043413688893,29.76550172605349],[-102.3850461875865,29.77106372336083],[-102.3900938995232,29.78270801101298],[-102.3775593626074,29.80047407731595],[-102.3631106888968,29.83140275592682],[-102.364907391343,29.84554720609411],[-102.3520715918881,29.8530413039526],[-102.3410319268525,29.86956756441394],[-102.327223478902,29.86272519865259],[-102.3216993730291,29.87834943072315],[-102.3012802199625,29.87783822632826],[-102.2620814124812,29.85354375870733],[-102.2332775245876,29.84860928935517],[-102.2214183641998,29.84040562676168],[-102.2000007385363,29.83703745772086],[-102.1874069338487,29.84887779970944],[-102.1596189803247,29.81442392238104],[-102.1435761098741,29.80357962274781],[-102.1167291715633,29.79372967576479],[-102.0494581735515,29.78534079456283],[-102.0347818765358,29.80432957513161],[-102.0140407073923,29.79766729075882],[-102.0005994205381,29.80530485202008],[-101.9847199416181,29.79850117275939],[-101.9817430765963,29.81496590252682],[-101.9653003961347,29.81102163262359],[-101.9616213090327,29.79816276924576],[-101.9129623263632,29.7976288912142],[-101.8748342057854,29.79407752373138],[-101.8506989228999,29.80759803850111],[-101.8359745219938,29.79234632406235],[-101.8257382806915,29.80451207113872],[-101.8036121827712,29.77982894931275],[-101.7709186014484,29.78768713770667],[-101.7382018450134,29.77256651605248],[-101.7206931378763,29.77612190277375],[-101.709150638072,29.76183362039527],[-101.6893126091172,29.77104534459979],[-101.6806100645837,29.7604767064396],[-101.6614649921521,29.77125849381106],[-101.6523241198488,29.75875320475244],[-101.6358061955452,29.758017088491],[-101.6288384797423,29.77030500269442],[-101.6065472577473,29.77413009350111],[-101.5748613393771,29.76927601619966],[-101.5747199001723,29.78817131395418],[-101.5678319032819,29.79903358778008],[-101.5404240730785,29.80886174816411],[-101.5352589900119,29.7991979945904],[-101.5395284568528,29.76055169212258],[-101.5035741876824,29.7637750820819],[-101.4794623840963,29.783472845084],[-101.4542033541914,29.78630626322124],[-101.4510429158096,29.75329809021069],[-101.416027916694,29.75041361074031],[-101.396439246959,29.73392814615618],[-101.3986971204216,29.71560707954395],[-101.3740332594443,29.70216527146238],[-101.3734603811325,29.67968266028569],[-101.3649378970142,29.66785635894551],[-101.3485696478725,29.66130887104537],[-101.3173171684476,29.66011260061797],[-101.3017505091567,29.65095928201577],[-101.3005021161544,29.63793873397588],[-101.311319761073,29.61447316680272],[-101.3105218065929,29.5868824471662],[-101.2968764622278,29.57222818182308],[-101.2783321708436,29.58896201706354],[-101.2805410375743,29.6181511088083],[-101.2683107539914,29.62980286768074],[-101.25275063656,29.62637800425725],[-101.2522736559007,29.60391150519642],[-101.2432096368911,29.56998019092114],[-101.2583329820333,29.54276552298161],[-101.2557692550697,29.52089077847549],[-101.2382489847789,29.52360334410014],[-101.1855522580365,29.51800185529609],[-101.1751221847736,29.51042286392631],[-101.1476330292124,29.47446149876937],[-101.1298064472409,29.47668707120513],[-101.1130947985384,29.47006423499758],[-101.0930755558439,29.47322664691718],[-101.0618123437121,29.46424552837134],[-101.057003761832,29.44075423229813],[-101.0430388482242,29.42915503832618],[-101.0368288551023,29.4062271562635],[-101.0205649656303,29.39245529102876],[-101.0050358141144,29.36544031063562],[-100.950433684284,29.34801689476265],[-100.9398504450884,29.33261895804229],[-100.9178259698024,29.32447706603438],[-100.8978947469702,29.30928796684396],[-100.8864355670299,29.3077751478743],[-100.8797391178435,29.28186181744407],[-100.8544377689164,29.27532580181495],[-100.8334903865815,29.26124279031427],[-100.818180739312,29.26425353547531],[-100.7942704696797,29.2421003322775],[-100.7738810180437,29.21027495942376],[-100.7662489252076,29.18489530696402],[-100.7750046053717,29.17484138839221],[-100.7635086742915,29.16010441143694],[-100.7428663210975,29.15273034265007],[-100.7378280936324,29.13902448814407],[-100.708363055786,29.1187371595999],[-100.6844067579495,29.11139456766108],[-100.6736134674755,29.09998268013288],[-100.6644223198122,29.07468146009754],[-100.6636691746984,29.05054866330335],[-100.6563099966303,29.0181005165352],[-100.6456665169906,28.98789840524609],[-100.6515310963175,28.94572092143414],[-100.6396470903341,28.92866061461697],[-100.632564656519,28.90374426796732],[-100.6225395612917,28.9101925088564],[-100.6040854585504,28.88888438352297],[-100.5905523476495,28.88355065087153],[-100.5915322778655,28.86463549255291],[-100.5729049478859,28.84796948524697],[-100.5769204123759,28.83677835819909],[-100.5466710769831,28.82545252153536],[-100.5358390718651,28.80596016928734],[-100.5377657389123,28.78009644424252],[-100.5073704342384,28.74041760321193],[-100.5100177148073,28.69136678997103],[-100.5001772041228,28.66166815679531],[-100.4796244453217,28.65560709661523],[-100.4614671377156,28.6408396133899],[-100.4467098257195,28.64204405844755],[-100.4471569446324,28.609227029814],[-100.4318881098645,28.59771043404291],[-100.3986938192532,28.58537602292263],[-100.3972899709987,28.57531618379625],[-100.410864015029,28.55285632946125],[-100.4033678890245,28.53366302258935],[-100.3885402601449,28.51539898647503],[-100.3375636422144,28.50156534095795],[-100.3443688338431,28.48615756787672],[-100.366524997958,28.47310054613012],[-100.3407956429993,28.44893554831869],[-100.335913031658,28.43013702030524],[-100.3494818862225,28.40340046912693],[-100.3429313797382,28.38715774846846],[-100.3187586799162,28.36002227455358],[-100.3141061238509,28.34554988367295],[-100.2911830787548,28.32001263172944],[-100.2861591422373,28.30746188558021],[-100.2942439801873,28.28357450864349],[-100.2894022027074,28.27378855956849],[-100.2565894205178,28.23929419711033],[-100.2247863070516,28.23574627965762],[-100.2170728567828,28.22576952020547],[-100.2113649003614,28.19393181782673],[-100.1738064419782,28.17890515543671],[-100.1591194054958,28.16768862271746],[-100.1406824659234,28.16800754385195],[-100.1200665084962,28.15584925312903],[-100.0901995757688,28.14832745800445],[-100.0678069402643,28.11432836651248],[-100.0564737521855,28.10475187318459],[-100.0530030216074,28.0850524095548],[-100.0283513728994,28.07313373853517],[-100.0167782783419,28.0618948704217],[-100.0076413207512,28.02146293584902],[-99.9904922577167,27.99361654243269],[-99.96017203677658,27.98333470840733],[-99.93379752123158,27.98183578852413],[-99.93883485579266,27.95216408190899],[-99.9175442794928,27.91791055063256],[-99.9006254202522,27.91244729490728],[-99.89394171792327,27.89297828058195],[-99.9027569600924,27.88194014733075],[-99.90131727113258,27.86440976459141],[-99.8815752922057,27.84961961321243],[-99.87588221297693,27.83766559208237],[-99.87778197139068,27.80146291044372],[-99.85075948991931,27.7939031730643],[-99.84322258736285,27.76986281894883]]]}}]}
```

**Comment by Pessimistress** [comment]
This should be fixed in 7.3.10.

**Comment by mohitshah9920** [comment]
Thank you for the fix!
